src/worker-type.js

```diff
@@ -779,6 +780,18 @@ WorkerType.prototype.determineCapacityChange = function(runningCapacity, pending
   return capacityChange;
 };
 
+WorkerType.prototype.declareWorkerType = function({queue}) {
+  // keep the worker around well over 24 hours, as we will likely re-declare
+  // at least that often
+  const expires = taskcluster.fromNow('36 hours');
+
+  return queue.declareWorkerType(this.provisionerId, this.workerType, {
+    stability: 'stable',
+    expires,
+    description: `${this.description}\n\nOwner: ${this.owner}`,
```

Comment:
not blocking this review, but maybe an owner field would be merited in the queue method?

Reply:
Yeah, I was between that and removing the owner field from the provisioner's WorkerTypes table, and this seemed like a compromise :)

Reply:
(since none of the owner fields contain anything especially meaningful)